android/app/src/main/java/com/zulipmobile/sharing/SharingModule.kt

```diff
@@ -12,6 +13,7 @@ internal class SharingModule(reactContext: ReactApplicationContext)
     @ReactMethod
     fun getInitialSharedContent(promise: Promise) {
         promise.resolve(initialSharedData)
+        initialSharedData = null
     }
```

Comment:
This is certainly plausible code now. I wonder if it's clearly the behavior we really want, though. Why are we ending up trying twice to get the initial shared data? What's the situation where that happens?